korean-air/src/components/mainPage/Promotion.tsx

```diff
@@ -32,3 +35,10 @@ const PromotionBox = styled.section`
     z-index: 1;
   }
 `;
+
+const PromotionCardBox = styled.div`
```

Comment:
```suggestion
const PromotionCardBox = styled.div<{$scroll : boolean}>`
```

쏘영이가 PR에 올려준 IntrinsicAttributes 오류는 바로바로 타입스크립트인데 타입을 지정해주지 않아서 발생하는 오류입니다!! 드디어 ! 타스를 적용해줄 때가 온거죵 ㅎㅎ 저 오류를 만나면 아!내가 타입 지정 안해줬구나 생각하고 빠르게 쇽삭 타입을 추가해주면 된답니당 

특히 이렇게 styled-components에게 props를 전달하는 경우엔 위처럼 평범하게 `변수명 : 타입` 형태로 지정하는 것이 아닌, `<{ prop명 : 타입}>` 이렇게 대괄호를 사용해줘야합니다!!!

Reply:
[관련 블로그!](https://thsd-stjd.tistory.com/134)

Reply:
헐헐 당장 추가할게요!!! 타스,,,어렵다